walletrpc/compact_formats.proto

```diff
@@ -21,6 +21,8 @@ message CompactBlock {
     uint32 time = 5;            // Unix epoch time when the block was mined
     bytes header = 6;           // (hash, prevHash, and time) OR (full header)
     repeated CompactTx vtx = 7; // zero or more compact transactions from this block
+    uint32 saplingCommitmentTreeSize = 8; // the size of the Sapling note commitment tree as of the end of this block
```

Comment:
I didn't realize there were multiple versions of this .proto file around; there's also one in librustzcash. We should perhaps figure out a way to have a single source of truth.